website/content/api-docs/secret/databases/cassandra.mdx

```diff
@@ -42,23 +42,35 @@ has a number of parameters to further configure a connection.
 - `insecure_tls` `(bool: false)` – Specifies whether to skip verification of the
   server certificate when using TLS.
 
-- `tls_server_name` `(string: "")` – Specifies the name to use as the SNI host when 
+- `tls_server_name` `(string: "")` – Specifies the name to use as the SNI host when
   connecting to the Cassandra server via TLS.
-  
+
 - `pem_bundle` `(string: "")` – Specifies concatenated PEM blocks containing a
   certificate and private key; a certificate, private key, and issuing CA
-  certificate; or just a CA certificate.
-
-!> **Known Issue:** There is a known issue when using `pem_bundle` with only a CA (no client certificate & key)
-   where Vault will not parse the CA certificate correctly. To work around this, use `pem_json` with the
-   following structure: `{"ca_chain": ["-----BEGIN CERTIFICATE-----\nMIIEFjC...FNYakP7I\n-----END CERTIFICATE-----"]}`
-   Also make sure the PEM data is properly JSON encoded with `\n` instead of newlines.
+  certificate; or just a CA certificate. Only one of `pem_bundle` or `pem_json` can be specified.
 
 - `pem_json` `(string: "")` – Specifies JSON containing a certificate and
   private key; a certificate, private key, and issuing CA certificate; or just a
   CA certificate. For convenience format is the same as the output of the
   `issue` command from the `pki` secrets engine; see
-  [the pki documentation](/docs/secrets/pki).
+  [the pki documentation](/docs/secrets/pki).  Only one of `pem_bundle` or `pem_json` can be specified.
```

Comment:
Should we have a note that the certs/key can't have newlines?

Reply:
I've added a note that the PEM data must be properly JSON encoded.